schema_cache.rst

```diff
@@ -160,4 +160,84 @@ To disable auto reloading, drop the trigger:
 
 .. code-block:: postgresql
 
-  DROP EVENT TRIGGER pgrst_watch
\ No newline at end of file
+  DROP EVENT TRIGGER pgrst_watch
+
+Finer-Grained Event Trigger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can refine the previous event trigger and only react to the events relevant to the schema cache. This also prevents unnecessary
+reloading when creating temporary tables(``CREATE TEMP TABLE``) inside functions.
+
+.. code-block:: postgresql
+
+  -- watch create and alter
+  CREATE OR REPLACE FUNCTION pgrst_ddl_watch() RETURNS event_trigger AS $$
+  DECLARE
+    obj record;
+  BEGIN
+    FOR obj IN SELECT * FROM pg_event_trigger_ddl_commands()
```

Comment:
```suggestion
    cmd record;
  BEGIN
    FOR cmd IN SELECT * FROM pg_event_trigger_ddl_commands()
```

I think in this trigger `cmd` is better, because `pg_event_trigger_ddl_commands()` returns commands, not objects.